app/src/main/java/com/sopt/now/gs/data/api/service/GsHometownService.kt

```diff
@@ -31,4 +32,9 @@ interface GsHometownService {
     suspend fun getGspay(
         @Query("type") type: String
     ): Response<BaseResponse<ResponseReserveGspayDto>>
+
+    @GET("$API/$PRODUCTS")
+    suspend fun getCategory(
```

Comment:
p1 : 
```suggestion
    suspend fun getReserveCategory(
```
위의 Gspay도 ReserveGsPay로 바꿔주세요